packages/webview_flutter/webview_flutter/README.md

```diff
@@ -92,3 +92,28 @@ android {
 
 To use Material Components when the user interacts with input elements in the WebView,
 follow the steps described in the [Enabling Material Components instructions](https://flutter.dev/docs/deployment/android#enabling-material-components).
+
+## iOS
+
+### Limits Navigations To App Bound Domains
+Controls whether navigation should be limited to App-Bound Domains on iOS (iOS 14+ required).
+By default `limitsNavigationsToAppBoundDomains` is false.
+
+Caveats:
+
+  * Requires app-bound domains to be added to iOS Info.plist
+    * Example:
+        ```xml
+          <plist version="1.0">
+          <dict>
+            <key>WKAppBoundDomains</key>
+            <array>
+                <string>example1.com</string>
+                <string>example2.org</string>
+                ...
+            </array>
+          </dict>
+        ```
+  * `Array` of App-Bound Domains is limited to 10 domains
+  * Content supplied by the app through local files, data URLs and HTML String are always treated
+    as App-Bound Domains, and do not need to be listed.
```

Comment:
Please link to authoritative documentation of the OS feature rather than duplicating it here (where it could easily become outdated).